server/README.md

```diff
@@ -0,0 +1,39 @@
+# Dose Amigos Server Side Code
+
+## Build instructions
+Run `./gradlew clean build` to build and run tests.
+`clean` will delete the build folder and `build` will create a build folder.
+In `build/distributions`, a file called `server.zip` will be created that contains all dependency jars as well as compiled classes.
+This is what is added to the lambdas
+
+## Code Conventions
+All classes will be contained in a package associated with the feature it belongs to.
+For example `Med.java` would belong in the `info.doseamigos.med` package.
+
+The code will be written in Java 8, so that it's compatible with Amazon AWS Lambdas.
+
+We'll handle dependencies between classes using Google Guice: https://github.com/google/guice
+* Each feature package will contain a module
+* Each request handler will create an injector to use in its constructor.
+  * Note that the injector should include everything required for that handler class.
+
+The general structure should be as follows:
+* All Echo specific Code belongs in a Speechlet class.
+  * For example, `MedSpeechlet.java` would be a Speechlet for med adding/removing
+  * Follow https://developer.amazon.com/appsandservices/solutions/alexa/alexa-skills-kit/getting-started-guide for conventions.
+* All Rest endpoints should beling in a WebService class.
+  * Can have mutliple Handler methods for each http method
```

Comment:
My thinking here is that we can specify multiple handlers in the same class, as that seems possible with Amazon Lambda